cmd/antrea-agent/agent.go

```diff
@@ -629,6 +631,23 @@ func run(o *Options) error {
 			o.enableAntreaProxy)
 	}
 
+	var packetSamplingController *packetsampling.Controller
+	if features.DefaultFeatureGate.Enabled(features.PacketSampling) {
+		packetSamplingController = packetsampling.NewPacketSamplingController(
+			k8sClient,
+			crdClient,
+			serviceInformer,
+			endpointsInformer,
+			packetSamplingInformer,
+			ofClient,
+			ifaceStore,
+			networkConfig,
+			nodeConfig,
+			serviceCIDRNet,
```

Comment:
serviceCIDRNet is not a must-have config and may be incorrect. New code should use `serviceCIDRProvider` to get serviceCIDR

Reply:
OK, I will remove this for now, as the current implementation only support name based service not ip based.

Reply:
I will remove this. the current impl only supported name based service as target.